packages/instantsearch.js/src/templates/carousel/carousel.tsx

```diff
@@ -56,6 +56,8 @@ type CarouselTemplateProps<TObject extends Record<string, unknown>> = Pick<
     item?: CarouselUiProps<TObject>['itemComponent'];
   };
   cssClasses?: Partial<CarouselUiProps<TObject>['classNames']>;
+} & {
+  sendEvent?: CarouselUiProps<TObject>['sendEvent'];
```

Comment:
You can instead pick `'items' | 'sendEvent'` from `CarouselUiProps` on L53.

```suggestion
} & {
  sendEvent?: CarouselUiProps<TObject>['sendEvent'];
```

Reply:
We have tried this approach at first, but TS was throwing errors, so we found the workaround to pick `sendEvent` as optional 🤔

Reply:
Ah, well 😅, it's fine. I'll resolve these and you can merge whenever you're ready.